src/flexfix

```diff
@@ -47,5 +48,13 @@ for line in sys.stdin:
     line = re.sub(r'(#line \d+ ".*)_pretmp', r'\1', line)
     # Fix 'register' storage class specifier is deprecated and incompatible with C++17
     line = re.sub(r'register ', '', line)
+    # Fix use of 'int' instead of 'size_t' to match declaration of FlexInput and FlexOutput on Mac OS only
+    if platform.system() == "Darwin":
```

Comment:
Thanks fir making this.

If this can be made to work generally, without checking platform, that is preferred, as typically the problem is related to the flex output being bad independent of OS.  (Although the output might be bad only on one OS, matching the flex output should be sufficient.)  This helps defend us if MacOS changes it again, which they likely will.